pxr/usd/usdPhysics/wrapCollisionGroup.cpp

```diff
@@ -208,7 +208,7 @@ WRAP_CUSTOM {
         ;
 
     class_<UsdPhysicsCollisionGroup::CollisionGroupTable>("CollisionGroupTable")
-        .def("GetGroups", &_WrapGetGroups,
+        .def("GetCollisionGroups", &_WrapGetCollisionGroups,
```

Comment:
Might be worth adding a deprecation warning for GetGroups for a release cycle or two? Thanks!